core/store/migrations/0034_create_feeds_managers.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up34 = `
+CREATE TABLE feeds_managers (
+    id SERIAL PRIMARY KEY,
+	name VARCHAR (255) NOT NULL,
+	uri VARCHAR (255) NOT NULL,
```

Comment:
255 characters enough? Is there a reason to limit length at all?

Reply:
For a name and URI it will be enough. I can't see any use cases for it to be longer than that.